backend/src/bundles/files/file-management.service.ts

```diff
@@ -0,0 +1,44 @@
+import { type FileEntity } from '~/bundles/files/file.entity.js';
+import { type FilesRepository } from '~/bundles/files/file.repository.js';
+
+class FileManagementService {
+    private filesRepository: FilesRepository;
+
+    public constructor(filesRepository: FilesRepository) {
+        this.filesRepository = filesRepository;
+    }
+
+    private determineFileType(fileName: string): 'video' | 'photo' {
+        const extension = fileName.split('.').pop()?.toLowerCase();
+
+        if (!extension) {
+            throw new Error(
```

Comment:
Instead of default Error you should use HttpError with message and status property. See the example in auth.service